packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -117,216 +117,218 @@ export namespace CommonCommands {
         id: 'core.open',
     };
 
-    export const CUT = Command.toDefaultLocalizedCommand({
+    export const CUT = TypedCommand.toDefaultLocalizedCommand<[], void>({
         id: 'core.cut',
         label: 'Cut'
     });
-    export const COPY = Command.toDefaultLocalizedCommand({
+    export const COPY = TypedCommand.toDefaultLocalizedCommand<[], void>({
         id: 'core.copy',
         label: 'Copy'
     });
-    export const PASTE = Command.toDefaultLocalizedCommand({
+    export const PASTE = TypedCommand.toDefaultLocalizedCommand<[], void>({
         id: 'core.paste',
         label: 'Paste'
     });
 
-    export const COPY_PATH = Command.toDefaultLocalizedCommand({
+    export const COPY_PATH = TypedCommand.toDefaultLocalizedCommand<[uris: URI[]], void>({
         id: 'core.copy.path',
         label: 'Copy Path'
     });
 
-    export const UNDO = Command.toDefaultLocalizedCommand({
+    export const UNDO = TypedCommand.toDefaultLocalizedCommand<[], void>({
         id: 'core.undo',
         label: 'Undo'
     });
-    export const REDO = Command.toDefaultLocalizedCommand({
+    export const REDO = TypedCommand.toDefaultLocalizedCommand<[], void>({
         id: 'core.redo',
         label: 'Redo'
     });
-    export const SELECT_ALL = Command.toDefaultLocalizedCommand({
+    export const SELECT_ALL = TypedCommand.toDefaultLocalizedCommand<[], void>({
         id: 'core.selectAll',
         label: 'Select All'
     });
 
-    export const FIND = Command.toDefaultLocalizedCommand({
+    // ???
+    export const FIND = TypedCommand.toDefaultLocalizedCommand<[], void>({
         id: 'core.find',
         label: 'Find'
     });
-    export const REPLACE = Command.toDefaultLocalizedCommand({
+    // ???
+    export const REPLACE = TypedCommand.toDefaultLocalizedCommand<[], void>({
         id: 'core.replace',
         label: 'Replace'
     });
```

Comment:
I could not figure out if those have handlers?

Reply:
I believe we rely on monaco itself for the implementation:

https://github.com/eclipse-theia/theia/blob/1626004f4d35f462cc8f57ec33f252f70d5b9f3f/packages/monaco/src/browser/monaco-command.ts#L37-L43